README.md

```
@@ -56,6 +54,73 @@
+            CLI kubeprobes->>Usuário: Sai com status 1
+        end
+    end
+```
+
+## Requisitos
+
+- Go 1.13 ou superior
+- kubectl
+- Um cluster Kubernetes acessível
+
+## Uso
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)